py-utils/src/utils/msg_bus/analytical_example.py

```diff
@@ -0,0 +1,51 @@
+from message_bus.bus import Bus
+# from bus.callback import MyCallback
+from message_bus import Producer
+from message_bus import Consumer
+from message_bus.analytical_consumer import AnalyticalConsumer
+# from config import Config
+# from bus.topic import Topic
+from message_bus.message import Message
+
+from message_bus.utils import log_decorator
```

Comment:
Please reuse already existing logger in the cortx-utils

Reply:
removed log_decorator